scripts/globals/garrison_data.lua

```diff
@@ -70,20 +95,32 @@ xi.garrison.allyLooks =
             "0x0100040105102420053005400550BB6000700000"
         },
         {
-            804,
-            805
+            -- Voidwatch officer (17752374)
```

Comment:
Are these just guessed? If you put in an incorrect placeholder, things are 100x less likely to ever be re-addressed and fixed

Reply:
Update: I've seen the comment, and still disagree that this is a valid solution

Reply:
These are from the looks in the npc_list. I went and found npcs I liked and then ran:
`Select name, npcid, CONCAT("0x", HEX(look)) FROM npc_list WHERE name LIKE "Shiv%";`

Reply:
Can we make a TODO note that we need to capture the actual looks still from retail?

Reply:
Will do. I'll make an issue too.